web/README.md

```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
```

Comment:
do we want to keep the generated readme ?

Reply:
I can nuke it.